pkg/web/transactions.go

```diff
@@ -406,13 +406,11 @@ func (s *Server) SendEnvelopeForTransaction(c *gin.Context) {
 		return
 	}
 
-	// If the content requested is HTML (e.g. the web-front end), then redirect the user
-	// to the transaction detail page and set a cookie to display a toast message
+	// If the content requested is HTML (e.g. the web-front end), then
+	// respond with a 204 no content response and the front-end will handle the
+	// success message in the toast.
 	if c.NegotiateFormat(binding.MIMEJSON, binding.MIMEHTML) == binding.MIMEHTML {
-		detailURL, _ := url.JoinPath("/transactions", packet.Transaction.ID.String(), "info")
-		setToastCookie(c, "transaction_send_success", "true", detailURL, s.conf.Web.Auth.CookieDomain)
-
-		htmx.Redirect(c, http.StatusFound, detailURL)
+		htmx.Trigger(c, "transactionCompleted")
```

Comment:
Now that we're using `/send` for completing the transaction, it felt best to replace the redirect with a trigger.